.teamcity/_self/projects/WebApp.kt

```diff
@@ -681,3 +681,101 @@ object PreReleaseE2ETests : BuildType({
 		executionTimeoutMin = 20
 	}
 })
+
+object QuarantinedE2ETests: BuildType( {
+	id("Quarantined_E2E_Tests")
+	name = "Quarantined E2E Tests"
+	description = "E2E tests quarantined due to intermittent failures."
+	maxRunningBuilds = 1
+
+	artifactRules = """
+		logs.tgz => logs.tgz
+		screenshots => screenshots
+		trace => trace
+	""".trimIndent()
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	steps {
+		prepareEnvironment()
+		bashNodeScript {
+			name = "Run e2e tests"
+			scriptContent = """
+				shopt -s globstar
+				set -x
+
+				cd test/e2e
+				mkdir temp
+
+				export URL="https://wpcalypso.wordpress.com"
```

Comment:
@dpasque What do you think about running this on `wpcalypso`?

Reply:
I think that's ideal, as that's where the pre-release tests ultimately run!!